Task2.py

```diff
@@ -0,0 +1,84 @@
+from collections import deque
+
+
+def hexsum(x, y):
+    HEX_NUM = {'0': 0, '1': 1, '2': 2, '3': 3, '4': 4, '5': 5, '6': 6, '7': 7, '8': 8, '9': 9,
+               'A': 10, 'B': 11, 'C': 12, 'D': 13, 'E': 14, 'F': 15,
+               0: '0', 1: '1', 2: '2', 3: '3', 4: '4', 5: '5', 6: '6', 7: '7', 8: '8', 9: '9',
+               10: 'A', 11: 'B', 12: 'C', 13: 'D', 14: 'E', 15: 'F'}
+    result = deque()
+    transfer = 0
+
+    if len(x) > len(y):
+        x, y = deque(y), deque(x)
+    else:
+        x, y = deque(x), deque(y)
```

Comment:
предпочтительнее делать одно действие за раз. так легче читать. сначала сделать обмен переменных, потом изменить тип на deque.
```suggestion
    if len(x) > len(y):
        x, y = y, x
    x, y = deque(x), deque(y)
```